modules/preview-react/form-field/lib/FormFieldHint.tsx

```diff
@@ -1,18 +1,33 @@
 import React from 'react';
-import {createSubcomponent, ExtractProps, useTheme} from '@workday/canvas-kit-react/common';
-import {space} from '@workday/canvas-kit-react/tokens';
+import {createSubcomponent, ExtractProps} from '@workday/canvas-kit-react/common';
+import {system, brand} from '@workday/canvas-tokens-web';
+import {createStencil} from '@workday/canvas-kit-styling';
 
 import {useFormFieldHint, useFormFieldModel} from './hooks';
 import {Subtext} from '@workday/canvas-kit-react/text';
+import {mergeStyles} from '@workday/canvas-kit-react/layout';
+
+const formFieldHintStencil = createStencil({
+  base: {
+    margin: `${system.space.x2} 0 0`,
+  },
+  modifiers: {
+    error: {
+      error: {
+        color: brand.error.base,
+      },
+      alert: {},
+    },
+  },
+});
 
 export const FormFieldHint = createSubcomponent('p')({
   displayName: 'FormField.Hint',
   modelHook: useFormFieldModel,
   elemPropsHook: useFormFieldHint,
 })<Omit<ExtractProps<typeof Subtext, never>, 'size'>>(
   ({children, ...elemProps}, Element, model) => {
-    const theme = useTheme();
-
+    console.log(formFieldHintStencil);
```

Comment:
Feels it should be removed :)

Reply:
who put that there?! señor pickle

Reply:
[!señor pickle Ogurchik](https://img.freepik.com/premium-vector/vintage-illustration-pickles-wear-hat-with-mustache-hold-stick_540637-597.jpg)